engine/sql/cdn/001_create_index.sql

```diff
@@ -1,13 +1,22 @@
 -- +migrate Up
 
--- Create new table for user registrations
 CREATE TABLE IF NOT EXISTS "index" (
-  id VARCHAR(36) PRIMARY KEY,
-  created TIMESTAMP WITH TIME ZONE,
-  name VARCHAR(255) NOT NULL,
+  id VARCHAR(36) PRIMARY KEY, -- technical ID
+  created TIMESTAMP WITH TIME ZONE, -- creation date
+  last_modified TIMESTAMP WITH TIME ZONE, -- last modified date
+  hash BYTEA, --
```

Comment:
by convention we decided to name "cipher_*" all encrypted column.
so hash => cipher_hash